README.md

```diff
@@ -1124,6 +1124,20 @@ semantic_tokens = true
 </tr>
 <!-- nvim-semantic-tokens -->
 
+<!-- nvim-surround -->
+</tr>
+<tr>
+<td> <a href="https://github.com/kylechui/nvim-surround">nvim-surround</a> </td>
+<td>
+
+```lua
+nvim_surround = true
```

Comment:
Seems that you've documented this integration enabled by default, but I can't find anything enabling it. Besides, I would recommend you to disable it by default so that this PR won't break the appearance of someone's Neovim.